src/kayenta/stages/kayentaStage/kayentaStage.controller.ts

```diff
@@ -137,7 +146,6 @@ export class KayentaStageController implements IComponentController {
     this.selectedCanaryConfigDetails = await this.loadCanaryConfigDetails();
     this.populateScopeNameChoices();
     this.setMetricStore();
-    this.overrideScoreThresholds();
```

Comment:
I don't see a mention of this in your PR description, but I'm assuming this is leaving because we're not supporting score thresholds in canary configs anymore. We should remove them from the canary config edit UI.